fixtures/structure/structures_fixtures.py

```diff
@@ -0,0 +1,112 @@
+null = None
+
+query_structures = '''
+   {
+      allStructures {
+        webId
+        level
+        name
+        parentId
+        tag
+        locationId
+        position
+      }
+    }
+        '''
+
+expected_response_structures = {
```

Comment:
@rajeman `structures_query_response` or `expected_structures_query_response` would be more descriptive.